Kernel/Output/HTML/Templates/Standard/AdminState.tt

```diff
@@ -73,7 +73,7 @@
                     [% Translate("Attention") | html %]: [% Translate("Please also update the states in SysConfig where needed.") | html %]
                 </p>
                 <p class="FieldExplanation">
-                    [% Translate("See also") | html %]: <a href="https://doc.otrs.com/doc" target="_blank">https://doc.otrs.com/doc</a>
+                    [% Translate("See also") | html %]: <a href="https://doc.znuny.org/doc" target="_blank">https://doc.znuny.org/doc</a>
```

Comment:
@tipue-dev Please use https://doc.znuny.org/manual/ to point to the new documentation